### ISSUE TITLE
void type with use-core should be core::ffi::c_void

### ISSUE THREAD

**Issue by hellow554** [rust-lang/rust-bindgen#1584]
### Input C/C++ Header

```C++
void *foo();
```

### Bindgen Invocation

```
$ bindgen input.h --use-core
```

### Actual Results

and/or

```rust
extern "C" {
    pub fn foo() -> *mut ::std::os::raw::c_void;
}
```
### Expected Results

```rust
extern "C" {
    pub fn foo() -> *mut ::core::ffi::c_void;
}
```

**Comment by emilio** [comment]
This one is easy to fix, and is one less dependency on `--ctypes-prefix`. Do you know since which rustc version is `core::ffi::c_void` stable?

**Comment by hellow554** [comment]
According to https://doc.rust-lang.org/core/ffi/enum.c_void.html since 1.1.0, so a long long time ago

**Comment by ktomsic** [comment]
This looks like it's still available (no one's commented here and I don't see any open pull requests to address it). Can I work on it?

**Comment by emilio** [comment]
Yes, please! That'd be great :)

**Comment by emilio** [comment]
Fixed by #1634, thanks @ktomsic :)
